test/services/websockets.test.js

```diff
@@ -35,6 +35,7 @@ describe('Web Socket Service', function () {
           }
         ];
         expect(store.getActions()).to.deep.equal(expectedActions);
+        done();
```

Comment:
The spot was @jackcarlisle's - I just `cherry-pick`-ed the commit as he's deep in other things.